qa/tasks/cephadm.py

```diff
@@ -343,6 +343,28 @@ def ceph_crash(ctx, config):
                     pass
 
 
+@contextlib.contextmanager
+def pull_image(ctx, config):
```

Comment:
does this speed up the job itself?

Reply:
Yes and no.  It was making the 'host add' loop take 30s per host, but I also changed that to issue all commands on bootstrap_remote, so not anymore... but that just pushed the pull later to whenever the first container is deployed on that host.  So, net, this should speed things up.